etna/search/views.py

```diff
@@ -1071,6 +1087,9 @@ def get_context_data(self, **kwargs):
         page_type_field.choices = sorted(replacement_choices, key=lambda x: x[1])
         page_type_field.choices_updated = True
 
+        kwargs["page_type"] = self.page_type
+        kwargs["page_title"] = self.page_title
+
         # Add custom variables to the return value
         context.update(
```

Comment:
Does this work for Website search? 
The `kwargs` are redundant at this point. 
Set these before `context = super().get_context_data(**kwargs)` in this method.